middleware/verify_privilege_token/main.go

```diff
@@ -0,0 +1,64 @@
+package verify_privilege_token
+
+import (
+	"fmt"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/rs/zerolog"
+	"golang.org/x/exp/slices"
+)
+
+type VerifyPrivilegeTokenInterface interface {
```

Comment:
Haha I think this is the same as `IVerifyPrivilegeToken` or whatever it was, we can go back to that, I think that's fine and it was a bit shorter.

Reply:
lol.. kind of hard naming this in a way that reflect what it is specifically without the I or interface in the variable name